recipes/openmc-plotter/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "openmc-plotter" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openmc-plotter-{{ version }}.tar.gz
+  sha256: 06e9e17f12f119e81b9fbf80cee9276cf3ec144a6be9f783b304646d357c32b4
+
+build:
+  entry_points:
+    - openmc-plotter=openmc_plotter.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win or py<36]
```

Comment:
What is the reason that this package cannot be noarch: python? It is noarch on PyPI.

Reply:
If a dependency is missing, it should still be noarch as the dependency can be made available later. Then, the user can install the package without you having to change anything.